daemons/fenced/fenced_commands.c

```diff
@@ -3541,11 +3546,11 @@ handle_reply(pcmk__client_t *client, xmlNode *request, const char *remote_peer)
  * \internal
  * \brief Handle a message from an IPC client or CPG peer
  *
- * \param[in] client      If not NULL, IPC client that sent message
- * \param[in] id          If from IPC client, IPC message ID
- * \param[in] flags       Message flags
- * \param[in] message     Message XML
- * \param[in] remote_peer If not NULL, CPG peer that sent message
+ * \param[in,out] client      If not NULL, IPC client that sent message
```

Comment:
`client` can be made const if you do the same for `handle_reply()`. All `handle_reply()` does is null-check and get the name.

Reply:
Sorry, missed the assignment to `.ipc_client`

Reply:
It's non-const when assigned to pcmk__request_t, which needs it that way